src/data/stream.py

```diff
@@ -2,116 +2,109 @@
-    while True:
-        _, lf = left.read()
-        _, rf = right.read()
-
-        # Remove mediapipe processing by commenting from here ---->
-
-        # recolor frame; required for mediapipe proc.
-        lf = cv2.cvtColor(cv2.flip(lf, 1), cv2.COLOR_BGR2RGB)
-        rf = cv2.cvtColor(cv2.flip(rf, 1), cv2.COLOR_BGR2RGB)
-
-        # detect hands
-        l_res = hands.process(lf)
-        r_res = hands.process(rf)
-
-        # return to og color to handle display
-        lf = cv2.cvtColor(lf, cv2.COLOR_RGB2BGR)
-        rf = cv2.cvtColor(rf, cv2.COLOR_RGB2BGR)
-
-        # add outline to hand
-        if l_res.multi_hand_landmarks and r_res.multi_hand_landmarks:
-            for hand_landmarks in l_res.multi_hand_landmarks:
-                mp_drawing.draw_landmarks(lf, hand_landmarks, mphands.HAND_CONNECTIONS)
-            for hand_landmarks in r_res.multi_hand_landmarks:
-                mp_drawing.draw_landmarks(rf, hand_landmarks, mphands.HAND_CONNECTIONS)
-
-        # to here ------>
-
-        # FPS tracking
-        new_frame_t = time()
-        fps = str(int(1 / (new_frame_t - old_frame_t)))
-        old_frame_t = new_frame_t
-
-        # uncomment for cli testing.
-        # print(f"FPS: {fps}")
-
-        # add fps counter to left frame.
-        cv2.putText(lf, fps, (7, 70), cv2.FONT_HERSHEY_SIMPLEX, 2, (100, 255, 0), 2, cv2.LINE_AA)
-
-        # show frames in separate windows. Purely for setup and debugging. We won't need to show any of
-        # images we capture during actual runtime.
-        cv2.imshow("left", lf)
-        cv2.imshow("right", rf)
-
-        # press q to exit.
-        if cv2.waitKey(1) & 0xFF == ord("q"):
-            break
-
-
-def setup():
-    def set_video_cap_attrs(cap):
-        # only keep two frames in buffer. Supposedly improves speed.
-        cap.set(cv2.CAP_PROP_BUFFERSIZE, cap_buffer_size)
-
-        # set capture resolution (low-res).
-        cap.set(cv2.CAP_PROP_FRAME_WIDTH, cam_width)
-        cap.set(cv2.CAP_PROP_FRAME_HEIGHT, cam_height)
-
-    # opencv setup
-    left = cv2.VideoCapture(0)
-    right = cv2.VideoCapture(1)
-    set_video_cap_attrs(left)
-    set_video_cap_attrs(right)
-
-    # more mediapipe setup
-    hands = mphands.Hands()
-
-    # verify attributes as expected.
-    def test_video_cap_attrs(cap):
-        res = (
-            int(cap.get(cv2.CAP_PROP_BUFFERSIZE)) == cap_buffer_size
-            and int(cap.get(cv2.CAP_PROP_FRAME_WIDTH)) == cam_width
-            and int(cap.get(cv2.CAP_PROP_FRAME_HEIGHT)) == cam_height
-        )
-        return res
-
-    assert (test_video_cap_attrs(left) and test_video_cap_attrs(right)) == True
-
-    return left, right, hands
-
-
-def teardown(left, right):
-    left.release()
-    right.release()
-
-    cv2.destroyAllWindows()
-
-
-def run():
-    left, right, hands = setup()
-
-    main(left, right, hands, old_frame_t, new_frame_t)
-
-    teardown(left, right)
-
+class InputStream:
```

Comment:
Global : Member functions should follow the docstring format.

```
def _create_file_impl(self, f_path: str, f_data) -> bool:
        """The implementation of create file.
        
        Args:
            f_path : the full path to save file data to.
            f_data : the data to save to file.
        Returns:
            True if the operation was a success; false otherwise.
            
        Note:
            If data is of type Numpy Array, the function will use OpenCV's image write function
            function to create a new image.
        """
      ....
```

Include a short description , any arguments passed to the function (if the function takes params), a description of what's returned (if anything is returned), and any major notes for use (as necessary).